apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Resolvers/GridAreaResolver.cs

```diff
@@ -28,37 +28,32 @@ public async Task<GridAreaStatus> CalculateGridAreaStatusAsync(
         var validTo = gridarea.ValidTo;
         var consolidation = await consolidationsLoader.LoadAsync(gridarea.Code).ConfigureAwait(false);
 
-        if (consolidation is null)
+        if (consolidation?.ConsolidateAt > DateTimeOffset.UtcNow)
```

Comment:
Suggestion: For another PR it might be a good idea to add a test that covers this status calculation

Reply:
Take a look at apps/dh/api-dh/source/DataHub.WebApi.Tests/Integration/GraphQL/BalanceResponsibilityAgreement/BalanceResponsibilityAgreementStatusTests.cs :-)

Reply:
Thanks for the pointer. Tried adding a test :)